hi-heavens/blog_api
# ch(connect-DB): connection to the mongoDB using Atlas

### What does this PR do?
* To successfully create a connection with the mongo DB using Atlas
* Setup different folder for the connection file

### Description of the task to be completed?
* Install the dependencies 
* Provide the necessary parameters to connect the db
* Setup an online mongo DB using Atlas

### How should this be manually tested?
* On the terminal, run the command npm start to start the server and confirm DB connection
* If successful, a message will be logged on the terminal
